### ISSUE TITLE
Swapping doesnt swap the name and info of the plugin

### ISSUE THREAD

**Issue by Kumicoo** [LightningSw/TeamInsightX#1]
Hey.

When someone swaps, the plugin doesnt swap with them.

My premade is mid.

he swaps with top.

my premade is still shown as mid

**Comment by LightningSw** [comment]
Yes, this is a bug. I will fix it soon, but before that, I will complete the multi-language support.
